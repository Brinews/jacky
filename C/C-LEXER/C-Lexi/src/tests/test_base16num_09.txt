
  project2x16
   1ABx16=1BBx16
