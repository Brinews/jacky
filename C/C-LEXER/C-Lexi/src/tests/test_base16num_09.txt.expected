{project2x16 , ID , 2}
{1ABx16 , BASE16NUM , 3}
{ , EQUAL , 3}
{1BBx16 , BASE16NUM , 3}
{ , END_OF_FILE , 5}
